README.md

```diff
@@ -1,2 +1,11 @@
-# Discussions about climate and Julia
-The repository is the central place for discussions about possible options and implementation for climate related packages.
+# Discussions About Climate And JuliaClimate
+
+This repository is a place for discussion of climate related packages and of the `JuliaClimate` organization.
+
+<img src="https://github.com/gaelforget/meta/raw/gfdev01/docs/juliaclimatelogo.png" width="50%">
+
+_**Organization logo design:** @cormullion & the @JuliaClimate community [code is here](https://bit.ly/3anhdMp)_
```

Comment:
Since it is already a hyperlink, why not just use the actual URL?

https://gist.github.com/gaelforget/4744a63b86f74362a71a89d27de44c90

Also, to make sure the logo is reproducible, perhaps it would be good to add a Manifest.toml, or simply write the Luxor and Colors versions as a comment?

Reply:
"Interestingly", gist links are blocked at my office :/

Reply:
> Since it is already a hyperlink, why not just use the actual URL?

Done

> Also, to make sure the logo is reproducible, perhaps it would be good to add a Manifest.toml, or simply write the Luxor and Colors versions as a comment?

Thanks for suggesting this -- doing it here for now. Just reran the gist code with:

```
  [5ae59095] Colors v0.11.2
  [ae8d54c2] Luxor v1.7.0
```